### ISSUE TITLE
Hent data fra brønden

### ISSUE THREAD

**Issue by KiraH** [DBCDK/metakompasset#74]
I version 2 benytter man en del af de eksisterende DBC-emneord.

Eksisterer der allerede emneord og genrer på posten skal de vises i metakompasset.

**Comment by cboesgaard** [comment]
Hvordan skal det hænge sammen med tildelte tags og anbefalinger? Er der tænkt over det?

**Comment by KiraH** [comment]
Nej, pt har vi koncentreret os om metakompas delen, men der ville også have været noget at tage stilling til der.

Men I2020 har nu besluttet ikke at tage allerede tildelte emneord fra brønden og ind i metakompasset.

**Comment by BettinaAndersen** [comment]
Kommer ikke med i version 2
